### PR TITLE
fix hcc linking error caused by __fdividef

### PR DESCRIPTION
**Pull request by sunway513**
This commit fixes linking error with recent HIP updates on, thanks to @AlexVlx .

### CODE REVIEW

**Comment by ROCm-Apps-Test** [comment]
Can one of the admins verify this patch?

**Comment by whchung** [comment]
ok to test

**Comment by sunway513** [comment]
Not tested yet -- this is a perfect need from NV CI..

**Comment by whchung** [comment]
@parallelo It seems "ok to test" doesn't really trigger the test.... guess I'm not admin?

**Comment by parallelo** [comment]
Interesting.  Will investigate admin settings.  I thought it was set up to have all of us as admins.  

Jenkins:  okay to test

**Comment by parallelo** [comment]
Jenkins:  ok to test

**Comment by parallelo** [comment]
Let me check the Jenkins logs for the blocker..

**Comment by parallelo** [comment]
Jenkins: add to whitelist

**Comment by parallelo** [comment]
Jenkins logs show:
```
Pull request #25s author has been whitelisted
```

Jenkins:  test this please

**Comment by parallelo** [comment]
Okay, all set.  `Build triggered for merge commit.`

Next:  figure out why the group whitelist isn't working properly with this Jenkins plugin.

File: tensorflow/core/kernels/fused_batch_norm_op.cu.cc [hunk 1]
```diff
@@ -50,7 +50,7 @@ __global__ void InvVarianceToVarianceKernel(int nthreads, double epsilon,
                                             int sample_size, T* variance) {
   GPU_1D_KERNEL_LOOP(index, nthreads) {
     T inv_var = variance[index];
-    T var = __fdividef(1, inv_var * inv_var) - T(epsilon);
```
Comment (whchung) [inline tensorflow/core/kernels/fused_batch_norm_op.cu.cc]: `__fdividef()` has some magic tricks in CUDA as it appears. I'd recommend using #if / #else to switch logic between CUDA and ROCm before we figure this out.

**Pull request merged**